src/Chapter1.hs

```diff
@@ -631,9 +644,13 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 n =
+    let last = n `mod` 10
+        prelast = (n `mod` 100)
```

Comment:
This is the right idea 💡  Though you need to consider the case of the negative numbers 🙂